Deploy-CAM.ps1

```diff
@@ -2709,6 +2661,155 @@ function Write-Host-Warning() {
     Write-Host ("`n$message") -ForegroundColor Red
 }
 
+# Prompt update Radius Settings in the Keyvault
+function Set-RadiusSettings() {
+    Param(
+        [string]$VaultName
+    )
+    # Check current MFA settings
+    $isRadiusMfaEnabled = Get-AzureKeyVaultSecret `
```

Comment:
Okay, this is probably worth a bit of thought/discussion, but some of the complexity is that we're kind of keeping this key vault secret as a cached value which we sometimes want and sometimes don't. Maybe we should just always be asking.